src/axom/primal/geometry/Hexahedron.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
```

Comment:
@bmhan12 -- could you please run the "update copyright year" script to ensure that all files have the right year?